src/conditionals/programs_with_decision_points/hard/q15/summation.java

```diff
@@ -0,0 +1,59 @@
+package conditionals.programs_with_decision_points.hard.q15;
+/*
+Write a program that asks the user for two numbers, num1 and num2, less than 50.
+Then, determine the sum of both numbers and classify it using switch:
+If the sum is less than 10:
+Print "Very small sum."
+If the sum is between 10 and 19 inclusive:
+If num1 is even, print "Even-medium sum."
+Otherwise, print "Odd-medium sum."
+If the sum is greater than or equal to 20 inclusive:
+If num2 is a multiple of 3, print "Multiple-of-3 large sum."
+Otherwise, print "Large sum."
```

Comment:
```suggestion
Write a program that asks the user for two numbers, num1 and num2 that are less than 50. Then, use a switch statement to determine the response to the sum of both numbers. If the sum is less than 10 print "Very small sum.", if it is between 10 and 19 inclusive print either "Even-medium sum." or "Odd-medium sum." If the sum is larger and divisible by 3 print "Multiple-of-3 large sum." otherwise just print "Large sum."
```

Reply:
you also mention in this question that the numbers should be less than 50 but there's no code response to if the numbers are greater than 50

Reply:
I forgot to remove that part! Nice catch thank you